src/modules/dashboard/components/RaiseObjectionDialog/RaiseObjectionDialogForm.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { FormikProps } from 'formik';
+
+import Button from '~core/Button';
+import DialogSection from '~core/Dialog/DialogSection';
+import { Annotations } from '~core/Fields';
+import Heading from '~core/Heading';
+import {
+  StakingSlider,
+  StakingAmounts,
+} from '~dashboard/ActionsPage/StakingWidget';
+
+import { Colony } from '~data/index';
+
+import { FormValues } from './RaiseObjectionDialog';
+import styles from './RaiseObjectionDialogForm.css';
+
+const MSG = defineMessages({
+  title: {
+    id: 'dashboard.RaiseObjectionDialog.RaiseObjectionDialogForm.title',
+    defaultMessage: 'Raise an objection',
+  },
+  objectionDescription: {
+    id: `dashboard.RaiseObjectionDialog.RaiseObjectionDialogForm.objectionDescription`,
+    defaultMessage: `
+    You are about to make an objection to the motion. If fully staked,
+    it will immediately start a voting process to determine whether
+    the motion should pass. <a>Learn more.</a>`,
+  },
+  annotation: {
+    id: 'dashboard.RaiseObjectionDialog.RaiseObjectionDialogForm.annotation',
+    defaultMessage: 'Explain why you’re making this objection (optional)',
+  },
+  stakeButton: {
+    id: 'dashboard.RaiseObjectionDialog.RaiseObjectionDialogForm.stakeButton',
+    defaultMessage: 'Stake',
+  },
+});
+
+const OBJECTION_HELP_LINK = `https://colony.io/dev/docs/colonynetwork/whitepaper-tldr-objections-and-disputes#objections`;
+
+export interface Props extends StakingAmounts {
+  colony: Colony;
+  canUserStake: boolean;
+  cancel: () => void;
+}
+
+const RaiseObjectionDialogForm = ({
+  colony,
+  handleSubmit,
+  isSubmitting,
+  canUserStake,
+  values,
+  cancel,
+  ...props
+}: Props & FormikProps<FormValues>) => {
+  return (
+    <>
+      <DialogSection appearance={{ theme: 'heading' }}>
+        <Heading
+          appearance={{ size: 'medium', margin: 'none' }}
+          text={MSG.title}
+          className={styles.title}
+        />
+      </DialogSection>
+      <DialogSection>
+        <FormattedMessage
+          {...MSG.objectionDescription}
+          values={{
+            a: (chunks) => (
+              <a
+                href={OBJECTION_HELP_LINK}
+                target="_blank"
+                rel="noopener noreferrer"
+                className={styles.link}
+              >
+                {chunks}
+              </a>
+            ),
```

Comment:
We have a core component for this, `ExternalLink` so that you don't have to bother with `target` and `rel`